### PR TITLE
First pass of copyediting pet shop tutorial

### PR DESCRIPTION
**Pull request by bmmpxf**

### CODE REVIEW

**Comment by bmmpxf** [comment]
@DiscRiskandBisque Please take a look. Mainly stylistic edits, but also some typo fixes as well. No screenshots yet.

File: _site/public/tutorials/pet-shop.md [hunk 1]
```diff
@@ -4,484 +4,594 @@
 
 # Ethereum Pet Shop
 
-This series of tutorials will take you through building your first DApp--- a tracking system for a pet shop!
+This series of tutorials will take you through building your first dapp---an adoption tracking system for a pet shop!
```
Comment (gnidan) [inline _site/public/tutorials/pet-shop.md]: So we're not calling them `ðapps`? 😜 
([Ð wiki](https://en.wikipedia.org/wiki/Eth))

Comment (bmmpxf) [reply _site/public/tutorials/pet-shop.md]: If we did that, we'd have to start calling it Eþereum. ;)

**Comment by bmmpxf** [comment]
Thanks @DiscRiskandBisque !

**Pull request merged**